src/utp_stream.cpp

```diff
@@ -713,7 +713,7 @@ bool utp_incoming_packet(utp_socket_impl* s
 	, udp::endpoint const& ep, time_point receive_time)
 {
 	return s->incoming_packet(
-		span<std::uint8_t const>(reinterpret_cast<std::uint8_t const*>(p.data()), int(p.size()))
+		span<std::uint8_t const>(reinterpret_cast<std::uint8_t const*>(p.data()), p.size())
```

Comment:
What do you think about `{reinterpret_cast<std::uint8_t const*>(p.data()), p.size()}` and make this one line